package.json

```diff
@@ -26,7 +26,9 @@
     "axios": "^0.25.0",
     "dotenv": "^14.2.0",
     "markdownlint-cli": "0.30.0",
-    "readline-sync": "^1.4.10"
+    "readline-sync": "1.4.10",
+    "thorify": "1.6.2",
+    "web3": "1.7.3"
```

Comment:
we already have `@vechain/connex`, and the other scripts are also written with that framework. Is there a reason we are using web3/thorify instead of connex?

Reply:
+ 1. I'm a fan of using `@vechain/connex` but I can see a discussion being held here where we talk about which library we want to use. there's web3, ethers (can't use now but in the future, although I believe someone made a vechain wrapper) and connex.

Reply:
@xenide can you send me where `@vechain/connex` is being used to deploy contracts?

Reply:
can see this [telegram link] (https://t.me/VeChainDevCommunity/24439). It's a bit obscure but doable

Reply:
trying to pass constructor arguments through clauses, still no success